README.rst

```diff
@@ -237,20 +250,37 @@ the `MLP` layer. In TE this can be added by setting the `layer_type` to `decoder
           layer_type="decoder",
   )
 
-Contributing to Transformer Engine
-----------------------------------
+Integrations
+==================
 
-We welcome contributions to Transformer Engine. To contribute to TE and make pull requests,
-follow the guidelines outlined in the `<CONTRIBUTING.rst>`_ document.
+Transformer Engine has been integrated with several popular open-source DL frameworks such as:
 
-Useful Links
-------------
+* `Hugging Face Accelerate <https://github.com/huggingface/accelerate>`_ 
+* `MosaicML Composer <https://github.com/mosaicml/composer>`_ 
+* `Megatron-LM <https://github.com/NVIDIA/Megatron-LM>`_ 
+* `ColossalAI <https://github.com/hpcaitech/ColossalAI>`_ - Coming soon!
+* `Lightning <https://github.com/Lightning-AI/lightning/>`_ - Coming soon!
+* `Periflow <https://github.com/friendliai/periflow-python-sdk>`_ - Coming soon!
```

Comment:
Should we add t5x and pax as comming " in the work"?

Reply:
Good idea, I think we should mention DL framework support with JAX/Praxis coming soon. 

As for T5x and Paxml maybe we can include a model examples section and eventually link JAX Toolbox / upstream code